docs/helm-oauth.md

```
@@ -0,0 +1,141 @@
+```
+
+## Notes
+
+- All `HEMMELIG_AUTH_*` variables require both `_ID` and `_SECRET`
+to enable a provider, except the "Gereric" type.
+
+If you enable a provider and not include the required environment variables for it,
+the pod will fail to start with CreateContainerConfigError, with an event
+similar to the one below:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)